rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+
+var config = {
+  entry: 'src/index.js',
+  dest: 'dist/ReactOnClickOutside.js',
+  format: 'umd',
+  moduleName: 'onClickOutside',
+  exports: 'named',
+  external: ['react', 'react-dom'],
+  globals: {
+    react: 'React',
+    'react-dom': 'ReactDOM'
+  },
+  plugins: [
+    nodeResolve({
+      jsnext: true
+    }),
+    babel({
+      exclude: 'node_modules/**',
```

Comment:
please use `external-helpers` plugin

Reply:
Shouldn't we do that via a separate babel env then?

Reply:
that is certainly approach that could be done, but `external-helpers` in rollup's context are somewhat magical and that magic is tightly coupled with rollup, so I think its a good place to have it the rollup's config rather than `.babelrc`

doesnt matter so much anyway, just got used to doing it like this, its also what most libs ive seen do

Reply:
The plugin increased the bundle size from 11945 to 12048.